### PR TITLE
Refs #35189 -- Moved CSS details' cursor declaration to base.css.

### PR DESCRIPTION
**Pull request by nessita**
# Trac ticket number

ticket-35189

# Branch description
Split CSS change where the `cursor: pointer` CSS declaration is moved to the toplevel base.css file in preparation for landing PR #17910.

# Checklist
- [X] This PR targets the `main` branch. 
- [X] The commit message is written in past tense, mentions the ticket number, and ends with a period.
- [X] I have checked the "Has patch" ticket flag in the Trac system.

### CODE REVIEW

**Review by ngnpope — APPROVED** [review]
🪄

File: django/contrib/admin/static/admin/css/changelists.css [hunk 1]
```diff
@@ -159,7 +159,6 @@
     font-weight: 400;
     padding: 0 15px;
     margin-bottom: 10px;
-    cursor: pointer;
```
Comment (MHLut) [inline django/contrib/admin/static/admin/css/changelists.css:162]: Does this break the cursor for `#changelist-filter h3`?

I wasn't sure about this yet, hence I hadn't removed it in the other PR.

Comment (ngnpope) [reply django/contrib/admin/static/admin/css/changelists.css:162]: Oh, missed that 🤦🏻

Although, should the `<h3>` have a pointer? I see that there are `<a>` tags within the `<h3>` tags anyway...

Comment (nessita) [reply django/contrib/admin/static/admin/css/changelists.css:162]: I don't think so, could you share a bit more what's your thinking?

The H3 under `#changelist-filter` looks like this with this change:
image

Comment (nessita) [reply django/contrib/admin/static/admin/css/changelists.css:162]: Good point Nick on why those are headers, initially I thought it was so screen readers announce that following the hierarchy:

```
H2 Filters -> H3 Show Counts
          |-> H3 By Author
...
```

But the details/summary in that sidebar are not using H3 :thinking:

Comment (nessita) [reply django/contrib/admin/static/admin/css/changelists.css:162]: As far as my testing goes, this does not break the existing `details summary` in the changelist filter, so I'll land and continue the work in PR #17910.

**Pull request merged**